core/uploadDiveLogsDE.cpp

```diff
@@ -246,18 +246,18 @@ void uploadDiveLogsDE::uploadDives(const QString &filename, const QString &useri
 	reply = manager()->post(request, multipart);
 
 	// connect signals from upload process
-	connect(reply, SIGNAL(finished()), this, SLOT(uploadFinished()));
+	connect(reply, SIGNAL(finished()), this, SLOT(slot_uploadFinished()));
```

Comment:
idiomatically I think I would prefer `uploadFinishedSlot()`, but that's trivial to change later, so this is fine.

Reply:
done for all slots.

Reply:
corrected